src/Frontend/Themes/Bootstrap4/Modules/Blog/Layout/Templates/Archive.html.twig

```diff
@@ -0,0 +1,59 @@
+{#
```

Comment:
As long as the core is not running on BS4, basically everything has to be overwritten by the theme templates, but I'll make sure they already extend & overwrite the blocks.